scene/gui/tree.cpp

```diff
@@ -2799,21 +2799,17 @@ int Tree::propagate_mouse_event(const Point2i &p_pos, int x_ofs, int y_ofs, int
 				return -1;
 			}
 
-			if (select_mode == SELECT_MULTI && p_mod->is_command_or_control_pressed() && c.selectable) {
-				if (!c.selected || p_button == MouseButton::RIGHT) {
-					p_item->select(col);
-					emit_signal(SNAME("multi_selected"), p_item, col, true);
-					emit_signal(SNAME("item_mouse_selected"), get_local_mouse_position(), p_button);
-
-					//p_item->selected_signal.call(col);
+			if (c.selectable) {
+				if (select_mode == SELECT_MULTI && p_mod->is_command_or_control_pressed()) {
+					if (c.selected && p_button != MouseButton::RIGHT) {
```

Comment:
```suggestion
					if (c.selected && p_button == MouseButton::LEFT) {
```
This condition was a bit confusing, so you could change it.

Reply:
Done